externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/SparkOperation.scala

```diff
@@ -259,13 +259,13 @@ abstract class SparkOperation(session: Session)
 
   override def shouldRunAsync: Boolean = false
 
-  protected def arrowEnabled(): Boolean = {
-    resultFormat().equalsIgnoreCase("arrow") &&
+  protected def arrowEnabled: Boolean = {
```

Comment:
`()` is removed since it's a pure function w/o side effects, this change addresses the IDEA warnings.

Reply:
@cfmcgrady